mlflow/sagemaker/container/__init__.py

```diff
@@ -61,9 +69,13 @@ def _serve():
     if pyfunc.FLAVOR_NAME not in m.flavors:
         raise Exception("Only supports pyfunc models and this is not one.")
     conf = m.flavors[pyfunc.FLAVOR_NAME]
+    model_py_version = None
+    if pyfunc.PY_VERSION in conf:
+        model_py_version = conf[pyfunc.PY_VERSION]
+
     bash_cmds = []
     if pyfunc.ENV in conf:
```

Comment:
1. Refactored so that creation only occurs in one place. We obtain a `CondaEnv` object first; this object specifies the environment's name, configuration file (if applicable) and python version (if applicable).

2. I think we should be able to parse the configuration file yaml and check for the version. If it's not specified, we can use the model's version. I know this differs from our offline discussion - how do you feel about that approach?

Reply:
Implemented the configuration modification described in (2)!